js/leo.js

```diff
@@ -257,6 +273,12 @@ function shutdown() {
     ros.close();
 }
 
+function defaultVideoSrc() {
+    if(typeof robot_namespace == 'undefined')
+        video.src = "http://" + robot_hostname + ":8080/stream?topic=/camera/image_raw&type=ros_compressed";
+}
```

Comment:
Here would be a good place to unsubscribe from the `robot_namespace` topic

Reply:
Should we unsubscribe the topic, only when we couldn't get robot namespace, or every time, as the `robot_namespace` topic is latched (should i unsubscribe inside, or outside the `if` statement)?

Reply:
Actually, I think it would be better to do at the start of this function

Reply:
Ok, done.